src/tdhLoop/tdh.ts

```
@@ -557,22 +555,27 @@ function hasSeasonSet(
   return seasonMemes.length === season.count;
 }
 
-function calculateMemesBoostsCardSets(cardSets: number) {
+function calculateMemesBoostsCardSets(cardSets: number, seasons: MemesSeason[]) {
```

ESLint

[error] Replace `cardSets:·number,·seasons:·MemesSeason[]` with `⏎··cardSets:·number,⏎··seasons:·MemesSeason[]⏎`

(prettier/prettier)